### PR TITLE
4 7 0 prep

### PR DESCRIPTION
**Pull request by nkoenig**
# 🎈 Release

Preparation for 4.7.0 release.

Comparison to 4.6.0: https://github.com/ignitionrobotics/ign-gazebo/compare/ignition-gazebo4_4.6.0...ign-gazebo4

Needed by:
* https://github.com/ignitionrobotics/ign-launch/pull/104

## Checklist
- [X] Asked team if this is a good time for a release
- [X] There are no changes to be ported from the previous major version
- [X] No PRs targeted at this major version are close to getting in
- [X] Bumped minor for new features, patch for bug fixes
- [X] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,69 @@
...
+1. Remove bounding box when model is deleted 
+    * [Pull Request 675](https://github.com/ignitionrobotics/ign-gazebo/pull/675)
+
+1. Cache link poses to improve performance.
+    * [Pull Request 669](https://github.com/ignitionrobotics/ign-gazebo/pull/669)
+
+1. Check empty world name in Scene3d.
+    * [Pull Request 662](https://github.com/ignitionrobotics/ign-gazebo/pull/662)
+
+1. All changes up to 3.8.0.
```
Comment (jennuine) [inline Changelog.md:65]: Is this needed or is it missing a PR link?

**Review by jennuine — APPROVED** [review]
LGTM just one minor comment on the changelog

File: Changelog.md (hunk 1 above)
Comment (nkoenig) [reply Changelog.md:65]: This is referring to the released features in version 3.8.0 that were forward ported.

Comment (jennuine) [reply Changelog.md:65]: Gotcha, thanks for clarifying

**Review by iche033 — APPROVED** [review]

**Pull request merged**